src/frontend/Canonicalize.ml

```diff
@@ -39,6 +39,8 @@ let rec replace_deprecated_expr
             ( StanLib
             , {name= rename_deprecated deprecated_distributions name; id_loc}
             , List.map ~f:(replace_deprecated_expr deprecated_userdefined) e )
+        else if String.is_suffix name ~suffix:"_cdf" then
+          CondDistApp (StanLib, {name; id_loc}, e)
```

Comment:
Good, this takes care of builtin `_cdf` functions. You should apply `replace_deprecated_expr` to `e`. Otherwise this unlikely code does not format correctly:
```stan
normal_cdf(normal_cdf(0, 1, 1), 0, 1)
```

User-defined `_cdf`s are also deprecated. That needs a similar check on line 57.